components/item/ItemHero.vue

```diff
@@ -95,6 +100,9 @@
       },
       downloadEnabled() {
         return this.rightsStatement && !this.rightsStatement.includes('/InC/');
+      },
+      showUserButtons() {
+        return Boolean(Number(process.env.ENABLE_XX_USER_AUTH)) && Boolean(Number(process.env.ENABLE_UNAUTHENTICATED_USER_BUTTONS));
```

Comment:
This won't show the button for logged in users now, add:
`|| (this.$store.state.auth && this.$store.state.auth.loggedIn));`
To bring it in line with this https://github.com/europeana/portal.js/blob/master/components/item/ItemPreviewCard.vue#L78

Reply:
oops, ok added